posts/renv-complications/index.qmd

```diff
@@ -0,0 +1,131 @@
+---
+title: "Things that can go wrong when using renv"
+author:
+  - name: "Hugo Gruson"
+    orcid: "0000-0002-4094-1476"
+date: "2023-09-26"
+categories: [R, reproducibility, renv]
+format:
+  html: 
+    toc: true
+---
+
+We use the [renv R package](https://rstudio.github.io/renv/) to ensure reproducibility of the training materials and the pipelines we are providing.
```

Comment:
```suggestion
Throughout epiverse, we use the [`renv` R package](https://rstudio.github.io/renv/) to ensure reproducibility of the training materials and the pipelines we are providing.
```

Reply:
I'm sticking to the tidyverse guidelines for package names. Adding the amended suggestion to still take into account the updated phrasing

```suggestion
Throughout the Epiverse project, we use the [renv R package](https://rstudio.github.io/renv/) to ensure reproducibility of the training materials and the pipelines we are providing.
```

Reply:
Today I learned! Thanks 😊